steelthread-0/walkthrough.md

```diff
@@ -15,123 +18,234 @@ The installation has already been performed on an OpenShift cluster as per the i
+- Click `Create`
 
 # Creating the integration
 
-- Open the Fuse Ignite console
-- Click on integrations
-- Click create integration
+- Stay in the Fuse Ignite console for this section
+- Click on `Integrations` and `Create Integration`
 - Click on the Enmaase connection
-- Click on the ``` Subscribe for messages ``` option
-- Under Destination Name add ``` work-queue/worker-updates ```
-- Choose Queue as the Destination Type
-- Click Next
-- Set the Specify Output Data Type as ``` Type specification not required ``` this is the default by clicking done
-- On the ``` Choose a Finish Connection ``` screen select the messaging booster connection you created
-- Choose ```Invoke URL ``` 
-- In the URL Path set the path to ```/api/data```
-- In the method select ```GET``` and click next
-- Again under Specify Input Data Type leave it as ``` Type specification not required ``` and click done
-- Click publish at the top right
-- Choose the integration name (steel thread)
+- Click on the ```Subscribe for messages``` option
+- Under Destination Name add ```work-queue/requests ```
+- Choose `Queue` as the Destination Type
+- Click `Next`
+- Set the `Select Type` as ```JSON Schema``` 
+- Add the following schema to the `Definition` field & click `Done`
+```
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"type": "object",
+	"properties": {
+		"type": {
+			"type": "string"
+		},
+		"stock": {
+			"type": "string"
+		}
+	}
+}
+```
+- On the ```Choose a Finish Connection``` screen select the `crud booster` connection you created
+- Choose ```Invoke URL``` 
+- In the URL Path set the path to ```/api/fruits```
+- In the method select ```POST``` and click next
+- Again under `Select Type` set it to ```JSON Schema``` 
+- Add the following schema to the `Definition` field & click `Done`
+```
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"type": "object",
+	"properties": {
+		"name": {
+			"type": "string"
+		},
+		"stock": {
+			"type": "string"
+		}
+	}
+}
+```
+
+- Next click the small plus between the two connections on the left hand side then click ```Add a step```
+- Click ```Data Mapper``` 
+- Map the `stock` property in the Source to the `stock` property in the Target by dragging from one side to the other. If done correct, a blue line will link the Source & Target.
+- Map `type` in the Source to `name` in the Target, again by dragging from one side to the other
+- Click `Done` at the top right
+
+## Add some Integration logging steps (optional)
+
+To add some more visibility we can add some logging steps to our integration.
+
+- On the left hand side inbetween the connections you will see small blue boxes with plus symbols. For each of these do the following:
+
+- click the blue plus symbol
+- click `Add a step`
+- choose `Log`
+- select ```Message Body```
+
+Once done we should now have 3 steps (5 if you added logging) in our integration. Click `Publish` in the top right.
+
+Name the integration ```steel thread```
+
```

Comment:
Need to click 'Publish' button again in the top right of the screen after setting the integration name ```steel thread```